config/development.env

```diff
@@ -6,8 +6,8 @@ CARD_CREATOR_PASSWORD=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAJ4wgZ
 
 #      Encrypted Values for API v0.2 and v0.3
 # ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
-ILS_CLIENT_KEY=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDIV8f7zUwdThoQ2E5AIBEIA3ywATnyN/WjzDGgyZEbdRvol37amRRqhOLnWyqy65kcmp3t4HpesPSGssBuGLIvkL4yCx99V9qA==
-ILS_CLIENT_SECRET=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHMwcQYJKoZIhvcNAQcGoGQwYgIBADBdBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDHA5aOhyQVpzqYNAnwIBEIAwxmiwst0+US+UpJYKCQtLuKLoR+fBIA2bNIpYWngeRLgGGjaY7/TmFZANotWrIAoy
+ILS_CLIENT_KEY=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDCiu+c44rmkAHZSzwgIBEIA3r3JbNsxtD1aVD+BVXmVWEnHlq8n8WVya7K3f6k6wnHQLZyATp+KfRAsShCvZoYK0lroQp0d5MA==
```

Comment:
I used the same creds for development and qa environment, I don't think they need to be separated, but please LMK if I shouldn't do that!

Reply:
that should be fine!